.gitignore

```diff
@@ -29,4 +29,6 @@ local-chart/
 # word counts
 /counts
 
-
+# Hub Chart generated dependencies
+hub-chart/Chart.lock
```

Comment:
s/hub/core/

Right now, this PR adds the file that you intend to exclude. We could fix this up in follow-on work.

Reply:
fixed